### DISCUSSION TITLE
Shlink Multiple Domain Support

### DISCUSSION THREAD

**Discussion by ibrahimdevs** [coollabsio/coolify#4474] [Q&A]
Hi, I setup Coolify mainly for Shlink. It works without a problem, but I want to support multiple domains. There is a documentation about it on Shlink website: https://shlink.io/documentation/advanced/multiple-domains/ , but I can't figure out how to do it on Coolify.

My use case:
Setup Shlink on Coolify for link.acme.com, it works. I want to support link.foo.com.
- I add A record to point link.foo.com to Coolify instance.
- I think I have to configure Shlink or Coolify to support link.foo.com but I don't know how to do?

*My coolify setup domain is acme.com and single server for coolify setup and shlink service.

Thanks,

**Comment by mikolajzawadzki** [comment]
i have the same problem. i also need support for managing multiple different domains. do i need any additional configuration in the reverse proxy for this?

**Comment by ibrahimdevs** [comment]
@mikolajzawadzki exact issue on my side too. For example I setup Wordpress via Coolify. It is working without a problem on coolify domain's subdomain. But there is SSL issue if I use another domain. I mean;
Coolify acme.com
My wordpress is working on: site.acme.com
There is SSL issue: foo.com so how can I use foo.com with Coolify? @andrasbacsai

**Comment by hnykda** [comment]
I just hit this too. It's really not possible? I am somewhat shocked.
